priv/gettext/es/LC_MESSAGES/dotcom.po

```diff
@@ -87,6 +87,9 @@ msgstr "Regístrate para T-Alerts"
 msgid "Boston Visitor's Guide to The T"
 msgstr "Guía del visitante de Boston para el T"
 
+msgid "Delays"
+msgstr "Delays"
```

Comment:
I know this isn't the point, but I suspect that this isn't right 😅

Reply:
i think it struggles with single capitalized words probably thinking they are proper nouns